bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -8,6 +8,7 @@ import SIPBridge from '/imports/api/audio/client/bridge/sip';
 import logger from '/imports/startup/client/logger';
 import { notify } from '/imports/ui/services/notification';
 import browser from 'browser-detect';
+import iosWebviewAudioPolyfills from './ios-webview-audio-polyfills';
```

Comment:
The polyfill utilitary could be moved to the `utils` folder since it'll be used by the other listen only implementation.

Reply:
I kept the js in this folder because this is a very specific fix, this was not thought to be reused. Where more do you think this polyfiil could be used?

Reply:
kurento-extension.js, for the SFU listen only implementation.